buildSrc/src/main/groovy/io/vantiq/extsrc/assy/tasks/AssemblyResourceGeneration.groovy

```diff
@@ -326,7 +349,7 @@ class AssemblyResourceGeneration extends DefaultTask {
         Map routeDoc = writeRoutesDocument(assemblyRoot, packageName, kamName, routesDocumentString)
         Map overviewDoc = (overview ? writeOverviewDocument(assemblyRoot, packageName, kamName, overview) : null)
         Map sourceDef = addSourceDefinition(assemblyRoot, packageName, kamName,
-            routeDoc.path.fileName.toString(), props)
```

Comment:
these are added to the source def'n as `additionalLibraries`, a feature that was already there (and worked!).